adaptor-rm-webapp/src/main/java/com/sample/rm/services/ServiceProviderService.java

```diff
@@ -126,12 +129,7 @@ public void getHtmlServiceProvider(@PathParam("serviceProviderId") final String
         // End of user code
 
         RequestDispatcher rd = httpServletRequest.getRequestDispatcher("/com/sample/rm/serviceprovider.jsp");
-        try {
-            rd.forward(httpServletRequest, httpServletResponse);
-        } catch (Exception e) {
-            e.printStackTrace();
-            throw new WebApplicationException(e);
-        }
+        rd.forward(httpServletRequest, httpServletResponse);
```

Comment:
I recall something about this block. Why did we decide to remove the whole try-catch block?